### PR TITLE
#9 fix roundoff error for lat & lng on German site for osm

### PR DESCRIPTION
**Pull request by philippze**
This fix is maybe not pretty, but works.

### CODE REVIEW

**Comment by bittner** [comment]
LGTM, the maximum value will always be below thousand. Hence this conversion seems valid.

File: djangocms_maps/static/djangocms_maps/js/mapbox.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ djangocms.Maps = {
         // latitute or longitute have precedence over the address when provided
         // inside the plugin form
         if (data.lat.length && data.lng.length) {
-            var latlng = { lat: parseFloat(data.lat), lng: parseFloat(data.lng) };
+            var latlng = { lat: parseFloat(data.lat.replace(',', '.')), lng: parseFloat(data.lng.replace(',', '.')) };
```
Comment (bittner) [inline djangocms_maps/static/djangocms_maps/js/mapbox.js]: Can you maybe put the code below, like so, and see if JavaScript is still happy?
```javascript
var latlng = {
    lat: parseFloat(data.lat.replace(',', '.')),
    lng: parseFloat(data.lng.replace(',', '.'))
 };
```

Comment (bittner) [reply djangocms_maps/static/djangocms_maps/js/mapbox.js]: @philippze Would you be willing to make that change? I'd merge the changes afterwards.

Comment (philippze) [reply djangocms_maps/static/djangocms_maps/js/mapbox.js]: Hallo @bittner , yes! I should find the time today.

**Comment by philippze** [comment]
Yes. It looks better like this. And I have tested that the code still works.

**Comment by bittner** [comment]
Thank you for your contribution, @philippze!

I hope you're happy as a user of this plugin. (Nice to know it's being used at all!)

**Pull request merged**